ios/Classes/FlutterArkitView+Handlers.swift

```diff
@@ -215,4 +215,17 @@ extension FlutterArkitView {
         }
         sceneView.scene.rootNode.removeAnimation(forKey: key)
     }
+
+    func onCameraEulerAngles(_ result:FlutterResult){
+        if let frame = sceneView.session.currentFrame {
+            let cameraEulerAngles = frame.camera.eulerAngles
+            var eulerAnglesArray: [Double] = [Double(cameraEulerAngles.x), Double(cameraEulerAngles.y), Double(cameraEulerAngles.z)]
+            let point = deserizlieVector3(eulerAnglesArray)
+            let res = serializeVector(point)
```

Comment:
I propose to create an instance of `SCNVector3` and call `serializeVector()` on it.